queue/queue.go

```diff
@@ -0,0 +1,303 @@
+package queue
+
+import (
+	"fmt"
+	"github.com/tarantool/go-tarantool"
+	"strings"
+	"time"
+)
+
+type Queue interface {
+	Put(data interface{}) (*Task, error)
+	PutWithOpts(data interface{}, cfg Opts) (*Task, error)
+	Take() (*Task, error)
+	TakeTimeout(timeout time.Duration) (*Task, error)
+	Drop() error
+	Peek(taskId uint64) (*Task, error)
+	Kick(taskId uint64) (uint64, error)
+	Statistic() (interface{}, error)
+}
+
+type Connection struct {
+	c    *tarantool.Connection
+	opts tarantool.Opts
```

Comment:
Yes, I thought about it, but did not dare to implement. Do you mind if I implement this?